ideascube/serveradmin/management/commands/csv2pkg.py

```diff
@@ -13,8 +16,29 @@
 Optional columns are "lang", "preview", "kind", "tags".
 Attributes "path" and "preview" (if provided) must be path
 (relative to csv directory) to existing files.
+
+csv2pkg also generates a yaml to add to the catalog.
+Some attributes are automically calculated from the generated package.
+Other attributes may be provided as command arguments.
 """
 
+def generate_catalog_yaml_metadata(path, _type, package_id, name,
+                                   description, language):
```

Comment:
A function for the only purpose of creating a dict seems like a bit much.

```python
print(yaml.safe_dump(
    {
        options['package_id']: {
            'name': options['name'],
            'description': options['description'],
            ...
        }
    }, default_flow_style=False)
```

?